client/components/forms/form-currency-input/style.scss

```diff
@@ -16,9 +16,17 @@ input[type="number"].form-currency-input {
 
 .form-text-input-with-affixes__prefix,
 .form-text-input-with-affixes__suffix {
+	display: flex;
+
 	&:hover .form-currency-input__select-icon {
 		color: var(--color-neutral-50);
 	}
+
+	&:focus-within {
+		z-index: 2;
```

Comment:
This is needed to pull the element above the input field, otherwise the focus only appears on 3 sides. I can add a comment if that's the practice here.

Reply:
That's a good catch. I confirmed it address what you mentioned. Thanks so much